packages/web/components/Modal/LoginModal/index.js

```diff
@@ -62,6 +64,9 @@ const LoginModal = ({ message: incomingMessage, redirectTo }) => {
 					<Link hover href="#">
 						{t('common:forgotPassword')}
 					</Link>
+					<Link hover href="#" onClick={() => openModal('emailConfirmation')}>
```

Comment:
Eu creio ser melhor verificamos o erro retornado na hora do login e se o erro for de confirmação de conta, nós abrimos um modal para que o usuário insira seu email solicitando a confirmação de conta novamente. No caso não precisariamos de um link específico para o usuário confirmar a conta. O que acha?